cmd/wallet-web/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@saeris/vue-spinners": "^1.0.8",
-    "@trustbloc-cicd/agent-sdk": "0.1.6-snapshot-84b797f",
+    "@trustbloc-cicd/agent-sdk": "^0.1.6-snapshot-1ce0ef1",
```

Comment:
You need to update package.json for BDD tests as well.
https://github.com/trustbloc/edge-agent/blob/ac276dd325bdce5f0272a9876b6a9ef725c0ea14/test/wallet-web/package.json#L6

Reply:
done